Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1007,6 +1007,49 @@ public extension Purchases {
         Self.configure(withAPIKey: apiKey, appUserID: appUserID, observerMode: false)
     }
 
+    /**
+     * Configures an instance of the Purchases SDK with a specified API key and
```

Comment:
this is that this section looks like when rendered in DocC:
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/3922667/233704451-bd263614-421e-477a-9051-f1d8a0a4ae66.png">

<img width="1073" alt="image" src="https://user-images.githubusercontent.com/3922667/233704432-28a39d67-8c84-418a-9598-c04be1d243ac.png">